core/variant/callable.cpp

```diff
@@ -35,6 +35,7 @@
 #include "core/object/script_language.h"
 #include "core/variant/callable_bind.h"
 #include "core/variant/variant_callable.h"
+#include "scene/main/node.h"
```

Comment:
This is problematic, `core` shouldn't depend on `scene`.
There are a couple exceptions currently that we need to fix, but in theory `core` shouldn't even know about Node.

Reply:
That's true, but RPC calls simply require `Node`, so unsure how to solve otherwise

Reply:
See #53295 for context.

Reply:
You can likely work it around with strings:
```cpp
Object *obj = ObjectDB::get_instance(ObjectID(object));
#ifdef DEBUG_ENABLED
if (!obj || !obj->is_class("Node")) {
	r_call_error.error = CallError::CALL_ERROR_INSTANCE_IS_NULL;
	r_call_error.argument = 0;
	r_call_error.expected = 0;
	return ERR_UNCONFIGURED;
}
#endif
r_call_error.error = CallError::CALL_OK;
return obj->call("rpcp", p_id, method, p_arguments, p_argcount);
```

Reply:
Sounds good, will look at that!